### PR TITLE
IDE-3166 update build.{user.name}.properties after initBundle

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by AndyWu2015** [comment]
jenkins retest

File: tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/action/InitBundleTaskAction.java [hunk 1]
```diff
@@ -43,9 +45,19 @@ protected void afterTask()
             if( bundlesLocation.toFile().exists() )
             {
                 ServerUtil.addPortalRuntimeAndServer( serverName, bundlesLocation, new NullProgressMonitor() );
+
+                IProject pluginsSDK = CoreUtil.getProject( "plugins-sdk" );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/action/InitBundleTaskAction.java]: They could have configured the plugins-sdk directory to be custom. Can we add the properties lookup here?

**Comment by AndyWu2015** [comment]
done

**Comment by AndyWu2015** [comment]
jenkins retest

**Comment by AndyWu2015** [comment]
jenkins retest

**Comment by AndyWu2015** [comment]
jenkins retest

**Comment by AndyWu2015** [comment]
this test fail rise by our update for m2e-wtp version. I am trying to fix it.

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-ide/pull/2144. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**